WooCommerce/WooCommerceTests/ViewRelated/Dashboard/Onboarding/StoreOnboardingViewModelTests.swift

```diff
@@ -563,3 +564,9 @@ private extension StoreOnboardingViewModelTests {
 
     final class MockError: Error { }
 }
+
+private extension Array where Element == StoreOnboardingTaskViewModel {
+    func containsOnlyPlaceHolderTasks() -> Bool {
+        map({ $0.task.type }).filter({ $0 != .launchStore }).isEmpty
```

Comment:
this feels a bit odd to me how a placeholder task is determined by checking if all the tasks are `launchStore` tasks. what happens if there is only one `launchStore` task from remote? maybe we don't need this check helper and can just assert the number of tasks since `isRedacted` state is more critical to show the placeholder state.

Reply:
> what happens if there is only one launchStore task from remote?

I planned to use this helper only for test purposes where we will be mocking the remote response. 

But, I understand your concerns. I removed this and checked for the number of tasks in https://github.com/woocommerce/woocommerce-ios/pull/9275/commits/630842309ef05436ee10718b9973a09a1756b40e 👍